android/app/src/main/cpp/app/organicmaps/MapManager.cpp

```diff
@@ -21,8 +21,11 @@
 #include <unordered_map>
 #include <vector>
 
+using namespace std::placeholders;
+
 namespace
 {
+using namespace storage;
```

Comment:
What were the errors in your case? Can you please add `storage::` where necessary instead of using `using`?